crypto/bls.go

```diff
@@ -93,7 +102,7 @@ const blsKMACFunction = "H2C"
 // tag is the domain separation tag, it is recommended to use a different tag for each signature domain
 func NewBLSKMAC(tag string) hash.Hasher {
 	// the error is ignored as the parameter lengths are in the correct range for kmac
-	kmac, _ := hash.NewKMAC_128([]byte(tag), []byte(blsKMACFunction), opSwUInputLenBLSBLS12381)
+	kmac, _ := hash.NewKMAC_128([]byte(tag), []byte(blsKMACFunction), minHashSizeBLSBLS12381)
```

Comment:
Similarly here, I think it is worth not letting the errors go silent. Leave the decision to you though, just a suggestion.

Reply:
I agree checking the error should be the default behaviour. 

In this case I ignored the error on purpose since the error is checking the input sizes and I made sure the inputs are chosen correctly. Since this is a low-level function that can be called frequently, I've decided to avoid the if else check and added the comment :
 `// the error is ignored as the parameter lengths are in the correct range for kmac`